### PR TITLE
pinned base requirements and added requirements for the manual separa…

### PR DESCRIPTION
**Pull request by grapesmoker**
This PR modifies `requirements/base.txt` and adds the file `requirements/manual.txt` to pin the `pip` dependencies of the project to their current versions. `manual.txt` is separated from the base requirements so that you have the option, should you choose to take it, of installing only the things you need to run the project and not the things you need to also display the manual.

## Additions
`requirements/manual.txt`

## Removals
Removed a few dependencies that did not seem to actually be used anywhere. 

## Changes
In addition to the above, reordered the requirements files to be in alphabetical order. Makes it easier to immediately see differences.

## Testing
`pip install -r requirements/base.txt` works.

## Review

@willbarton @richaagarwal @rosskarchner @chosak 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
+git+https://github.com/rosskarchner/govdelivery.git#egg=govdelivery
+html5lib==0.9999999
+icalendar==3.9.0
 lxml==3.6.0
 pbr==1.9.1
-django-compressor==2.0
\ No newline at end of file
+py==1.4.31
+python-dateutil==2.1
+pytz
```
Comment (willbarton) [inline requirements/base.txt]: Can we pin pytz?

File: requirements/test.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
 -r base.txt
 mock==1.3.0
-tox==2.1.1
```
Comment (willbarton) [inline requirements/test.txt]: Do we need tox in the base requirement and not test?

Comment (rosskarchner) [inline requirements/test.txt]: I don't think so?

Comment (chosak) [inline requirements/test.txt]: You only need tox if you're actually running the tests. Travis [uses](https://github.com/cfpb/cfgov-refresh/blob/master/.travis.yml#L34) `cfgov.settings.test` when running them.

File: requirements/base.txt (hunk 1 above)
Comment (grapesmoker) [inline requirements/base.txt]: I tried. It seems that pytz has this weird versioning that uses a YYYY.MM.DD format. Pip does not understand how to resolve this and therefore complains about it. I'm not sure what the right solution here is; it was previously left unpinned for, presumably, the same reason. Open to suggestions as to what to do.

Comment (rosskarchner) [inline requirements/base.txt]: I think a requirements file can point right to the tarball or wheel?  try replacing the pytz line with the URL:

https://pypi.python.org/packages/ba/c7/3d54cad4fb6cf7bf375d39771e67680ec779a541c68459210fcfdc3ba952/pytz-2016.6.1-py2.py3-none-any.whl#md5=735126cd1c1ddc824b7c94e091d7173e

**Comment by willbarton** [comment]
@grapesmoker can you move tox back to the test settings? Then I think we can get this merged.

**Comment by rosskarchner** [comment]
👍 after the change @willbarton mentions above

**Comment by grapesmoker** [comment]
So, I tried to add the tox requirement back into test, and this breaks Travis because apparently duplicating requirements Is Bad:

```Double requirement given: tox==2.3.1 (from -r ./requirements/test.txt (line 4)) (already in tox==2.3.1 (from -r ./requirements/base.txt (line 41)), name='tox')```

@willbarton @rosskarchner please advise

**Comment by rosskarchner** [comment]
Does anyone see a reason for it to be in base.txt? I feel like it can be removed there

**Comment by willbarton** [comment]
Yeah, we don't need tox in both places, and there's no reason for it to be in the base requirements.txt that I can see.

**Comment by willbarton** [comment]
:shipit:

**Comment by rosskarchner** [comment]
👍

**Pull request merged**